frontend/src/views/camp/Collaborators.vue

```diff
@@ -88,10 +92,16 @@ Displays collaborators of a single camp.
                     :my="0"
                     dense
                     vee-rules="required"
+                    :immediate-validation="true"
```

Comment:
I'm surprised that eslint / prettier don't have a strong opinion on this. I'd simplify this as just:
```suggestion
                    immediate-validation
```